api/cloud/cwpp/22-01/cwpp-home.mdx

```diff
@@ -29,6 +29,21 @@ To find the the version of Prisma Cloud Workload Protection that you're running:
 
    ![Console screenshot](img/compute-version.png)
 
+### View parameter descriptions
+
+The parameter descriptions are available for each endpoint. The body or query (wherever applicable) parameters are listed after the endpoint description.
+The response parameters are hidden under the label `Response` 200. 
+
+Click `>` to view hidden parameters.
+
+#### View API endpoint body parameters
+
+![Expand Body Parameters](../img/expandingbodyparameters.gif)
```

Comment:
It's best to store the images in the static/img folder. Can you move these and fix the references in this file and the 21-08 home.mdx?

You are referring to them as "img/file.gif" in the cwpp-home.mdx, but are using "../img/file.gif" in the versioned ones.